backend/controller/AccountController.ts

```diff
@@ -90,12 +100,12 @@ export class AccountController {
         }
         if (users.length > 1) {
             throw new LoernwerkError(
-                'ambiguous user details',
+                LoernwerkErrorMessages.AMBIGUOUS_USER_DETAILS,
                 LoernwerkErrorCodes.AMBIGUOUS_INFORMATION
             );
         } else if (users.length === 0) {
             throw new LoernwerkError(
-                'Mail/Name not matching an existing User',
+                LoernwerkErrorMessages.USER_NOT_FOUND,
```

Comment:
I dont think user not found is the right thing here, It should rather say something like `username or password is incorrect`